src/utils/handleError.tsx

```diff
@@ -0,0 +1,14 @@
+export const handleError = (err, callback) => {
```

Comment:
What's the purpose of this handleError function, as we already have errorHandlerContext?

Reply:
This is removed